spec/plaindate.html

```diff
@@ -391,9 +393,11 @@ <h1>Temporal.PlainDate.prototype.subtract ( _temporalDurationLike_ [ , _options_
         1. Let _balanceResult_ be ? BalanceDuration(_duration_.[[Days]], _duration_.[[Hours]], _duration_.[[Minutes]], _duration_.[[Seconds]], _duration_.[[Milliseconds]], _duration_.[[Microseconds]], _duration_.[[Nanoseconds]], *"days"*).
         1. Set _options_ to ? NormalizeOptionsObject(_options_).
         1. Let _overflow_ be ? ToTemporalOverflow(_options_).
-        1. Let _result_ be ? SubtractDate(_temporalDate_.[[ISOYear]], _temporalDate_.[[ISOMonth]], _temporalDate_.[[ISODay]], _duration_.[[Years]], _duration_.[[Months]], _duration_.[[Weeks]], _balanceResult_.[[Days]], _overflow_).
-        1. Assert: ! ValidateDate(_result_.[[Year]], _result_.[[Month]], _result_.[[Day]]) is *true*.
-        1. Return ? CreateTemporalDateFromInstance(_temporalDate_, _result_.[[Year]], _result_.[[Month]], _result_.[[Day]]).
+        1. Let _dateSubtractOptions_ be ! OrdinaryObjectCreate(%Object.prototype%).
+        1. Perform ! CreateDataPropertyOrThrow(_dateSubtractOptions_, *"overflow"*, _overflow_).
+        1. Let _constructor_ be ? SpeciesConstructor(_temporalDate_, %Temporal.PlainDate%).
+        1. Let _dateSubtract_ be ? Get(_calendar_, *"dateSubtract"*).
```

Comment:
There is no longer a Calendar.dateSubtract method, it became unnecessary when we changed the order in which fields were added/subtracted a while ago. This should use CreateTemporalDuration to instantiate a new duration which is the negative of the passed-in one, and pass that to Calendar.dateAdd.